lib/aptible/cli/agent.rb

```diff
@@ -80,7 +81,10 @@ def self.exit_on_failure?
 
       def initialize(*)
         nag_toolbelt unless toolbelt?
-        Aptible::Resource.configure { |conf| conf.user_agent = version_string }
+        Aptible::Resource.configure do |conf|
+          conf.user_agent = version_string
+          # conf.logger.tap { |l| l.level = Logger::INFO }
```

Comment:
This is nice for debugging and I kind of want to keep it in the code so it's discoverable.